### ISSUE TITLE
Hashtag dApp not pulling in production environment variables from platform.sh

### ISSUE THREAD

**Issue by swaylock** [hashtag-protocol/hashtag-protocol#225]
See https://app.hashtag-protocol.org for an example of most recent deployment that uses Nuxt. This deployment is failing to pull in the Platform.sh environment variables at runtime & apparently build time. 

Need to look at alternative ways to get the platform.sh environment variables into the dApp in the specific environment. Might be using this module https://github.com/platformsh/config-reader-nodejs

or perhaps some other solution.

https://github.com/nuxt/nuxt.js/issues/5100

I also saw a couple other leads that might be promising:

https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-runtime-config#publicruntimeconfig
https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-env#automatic-injection-of-environment-variables

Mind you, to get this to work with the SPA in platform.sh, I had to do this:

https://github.com/hashtag-protocol/hashtag-protocol/commit/10615bb48999cf3225bde64a8cf2d513b5620f29

Basically build the environment variables into a config file that is ingested by the SPA at runtime. It was the only solution that worked then, but I never liked it.

**Comment by swaylock** [comment]
This issue was resolved by an enhancement added by platform.sh to how it handled production environment variables.

https://docs.platform.sh/bestpractices/environment-build.html#devprod-environment-variables
